host/xtest/ffa_spmc_1000.c

```diff
@@ -0,0 +1,479 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2022, Arm Limited and Contributors. All rights reserved.
+ */
+#include <fcntl.h>
+#include <ffa.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+#include "include/uapi/linux/arm_ffa_user.h"
+#include "xtest_helpers.h"
+#include "xtest_test.h"
+
+#define FFA_DRIVER_FS_PATH	"/sys/kernel/debug/arm_ffa_user"
+#define SPMC_TEST_OK 0xaa
+#define INCORRECT_ENDPOINT_ID 0xffff
+#define NORMAL_WORLD_ENDPOINT_ID	0
+
+#define MEM_SHARE_HANDLE_LSB(x) ((x) & 0xffffffff)
```

Comment:
What is "LSB" short for?

Reply:
Added a comment

Reply:
bytes or bits? :-)
How about instead talking about low/high part of a handle? Using the abbreviations LSB and MSB may bring different expectations.

Reply:
Ack